### PR TITLE
docs: update specific for auto response status

### PR DESCRIPTION
**Pull request by AlbertAZ1992**
fix: https://github.com/koajs/koa/issues/1464

### CODE REVIEW

File: docs/api/response.md [hunk 1]
```diff
@@ -130,9 +130,18 @@ ctx.response.status = 204;
   - `Buffer` written
   - `Stream` piped
   - `Object` || `Array` json-stringified
-  - `null` no content response
+  - `null` || `undfined` no content response
+
+If `response.status` has not been set, Koa will automatically set the status to `200` or `204` depending on `response.body`. Specifically, if `response.body` has not been set or has been set as `null` || `undfined`, which means no content response, Koa will automatically set `response.status` to `204`. If you really want to send no content response with other status, you should override the `204` status as the following way:
```
Comment (miwnwski) [inline docs/api/response.md:135]: `undfined` misspelled.

Comment (miwnwski) [reply docs/api/response.md:135]: I would remove `which means no content response`, the reason to why null or undefined it set is ambiguous to Koa - but it interprets it as 204 No Content as you wrote.

Comment (miwnwski) [reply docs/api/response.md:135]: NIT: I would not use `||`'s for documentation text, rather use explicit `or`

Comment (AlbertAZ1992) [reply docs/api/response.md]: Thanks. That was updated. I also thought about `or` against `||` as definition listed above. Now i change to use `or` for documentation text.

**Pull request merged**